spec/requests/api/v1/game_nights/show_spec.rb

```diff
@@ -2,34 +2,66 @@
 
 RSpec.describe 'GET Game Night' do
   it 'gets the data about a game night' do
-    user_1 = create :user
-    user_2 = create :user
+    user1 = create :user
+    user2 = create :user
+    user3 = create :user
 
-    Friend.create(user_id: user_1.id, bud_id: user_2.id)
-    Friend.create(user_id: user_2.id, bud_id: user_1.id)
+    Friend.create(user_id: user1.id, bud_id: user2.id)
+    Friend.create(user_id: user2.id, bud_id: user1.id)
+    Friend.create(user_id: user1.id, bud_id: user3.id)
+    Friend.create(user_id: user3.id, bud_id: user1.id)
 
-    game_1 = create :game
-    game_2 = create :game
+    game1 = create :game
+    game2 = create :game
 
-    UserGame.create(user_id: user_1.id, game_id: game_1.id)
-    UserGame.create(user_id: user_2.id, game_id: game_2.id)
+    UserGame.create(user_id: user1.id, game_id: game1.id)
+    UserGame.create(user_id: user2.id, game_id: game2.id)
 
-    game_night = GameNight.create!(
-      user_id: user_1.id,
+    gamenight = GameNight.create!(
+      user_id: user1.id,
       name: 'Friday Fun Night',
       date: '1/15/2021',
       number_of_games: 2
     )
 
+    Invitation.create!(
+      status: 'accepted',
+      user_id: user2.id,
+      gamenight_id: gamenight.id
+    )
+
     Invitation.create!(
       status: 'pending',
-      user_id: user_2.id,
-      game_night_id: game_night.id
+      user_id: user3.id,
+      gamenight_id: gamenight.id
     )
 
-    get "/api/v1/game_nights/#{game_night.id}"
+    get "/api/v1/gamenights/#{gamenight.id}"
+
+    gamenightdata = JSON.parse(response.body, symbolize_names: true)
```

Comment:
FYI I think I moved all of this into a response checker.

Reply:
did you do that within the last hour? because I pulled from main before making any changes and I didnt see a checker for game night responses

Reply:
Ok I will adjust the test to use the checker